pages/common/R.md

```diff
@@ -0,0 +1,32 @@
+# R
+
+> R language interpreter.
+> More information: <https://www.r-project.org>.
+
+- Start an R interactive shell (REPL):
+
+`R`
+
+- Check R version:
+
+`R --version`
+
+- Start R in vanilla mode (i.e. blank session that doesn't save workspace at the end):
+
+`R --vanilla`
+
+- Execute a file:
+
+`R -f {{file.R}}`
```

Comment:
Perhaps this will read better?
```suggestion
`R -f {{path/to/file.R}}`
```

Also, is the file extension commonly `R` in uppercase? That's rather unusual.

Reply:
R files are usually written with the uppercase extension. For instance, these are some R scripts from R source code (note that they end in uppercase R): https://github.com/wch/r-source/tree/5a156a0865362bb8381dcd69ac335f5174a4f60c/src/library/base/R

Reply:
Thanks for clarifying!